client/blocks/reader-author-link/docs/example.jsx

```diff
@@ -9,16 +9,17 @@ import React from 'react';
 import ReaderAuthorLink from 'blocks/reader-author-link';
 import Card from 'components/card';
 
-export default React.createClass( {
+const ReaderAuthorLinkExample = () => {
 
-	displayName: 'ReaderAuthorLink',
+	const author = { URL: 'http://wpcalypso.wordpress.com', name: 'Barnaby Blogwit' };
 
-	render() {
-		const author = { URL: 'http://wpcalypso.wordpress.com' };
-		return (
-			<Card>
-				<ReaderAuthorLink author={ author }>Author site</ReaderAuthorLink>
-			</Card>
-		);
-	}
-} );
+	return (
+		<Card>
+			<ReaderAuthorLink author={ author }>{ author.name }</ReaderAuthorLink>
```

Comment:
Thanks @southp! That's a good point. I'll change it.

There's actually a case for `ReaderAuthorLink` not rendering its children at all, as `author.name` is always the value at the moment. I structured it this way to be consistent with other link components like `ExternalLink` and `ReaderSiteStreamLink`. Any preference one way or the other?